src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir-Book" ;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.searchBar {
+  display: block;
+  margin: 20px 0;
+}
+
+.Big-searchBar {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+}
+
+.Big-searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.Big-searchBar:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.search__swg {
+  margin: 24px 0 26px 25px;
+  position: absolute;
+}
+
+.input__field {
+  border: none;
+  margin-left: 62px;
+  margin-top: 25px;
+  margin-bottom: 23px;
```

Comment:
Don't use margins. If I wrote a long text into input - result will be like this - 
<img width="436" alt="Screenshot 2022-01-19 at 12 16 05" src="https://user-images.githubusercontent.com/32794096/150110775-4d170f66-0970-4636-99d5-2aefd31ce05e.png">